src/components/DayPicker.jsx

```diff
@@ -509,6 +517,12 @@ export default class DayPicker extends React.Component {
       translationValue: 0,
       nextFocusedDate: null,
       focusedDate: newFocusedDate,
+    }, () => {
+      // we don't want to focus on the relevant calendar day after a month transition
+      // if the user is navigating around using a mouse
+      if (this.state.withMouseInteractions && document.activeElement) {
```

Comment:
`document.activeElement` is the body by default. So when you blur, it gets set to the body. So this will end up blurring on the body every time. Probably not a big deal, but it is possible that this could kick off event listeners or other side-effects, depending on what has been registered on the page. Something like this might work? (not sure about browser support of HTMLBodyElement off the top of my head though)

```js
!(document.activeElement instanceof HTMLBodyElement)
```